process/transaction/shardProcess.go

```diff
@@ -136,14 +143,15 @@ func NewTxProcessor(args ArgsNewTxProcessor) (*txProcessor, error) {
 		scrForwarder:                   args.ScrForwarder,
 		signMarshalizer:                args.SignMarshalizer,
 		relayedTxEnableEpoch:           args.RelayedTxEnableEpoch,
+		relayedTxV2EnableEpoch:         args.RelayedTxV2EnableEpoch,
 		penalizedTooMuchGasEnableEpoch: args.PenalizedTooMuchGasEnableEpoch,
 		metaProtectionEnableEpoch:      args.MetaProtectionEnableEpoch,
 	}
 
 	log.Debug("shardProcess: enable epoch for relayed transactions", "epoch", txProc.relayedTxEnableEpoch)
 	log.Debug("shardProcess: enable epoch for penalized too much gas", "epoch", txProc.penalizedTooMuchGasEnableEpoch)
 	log.Debug("shardProcess: enable epoch for meta protection", "epoch", txProc.metaProtectionEnableEpoch)
-
+	log.Debug("shardTxProcessor: enable epoch for relayed transactions v2", "epoch", args.RelayedTxV2EnableEpoch)
```

Comment:
shardProcess: to keep the same format with the other logs above 
and 
txproc.relayedTxV2EnableEpoch instead args.RelayedTxV2EnableEpoch, to be sure that it is set inside struct

Reply:
done